src/components/modules/epics/ContributionsEpicArticleCountOptOut.tsx

```diff
@@ -0,0 +1,295 @@
+// --- Imports --- //
+
+import React, { useState } from 'react';
+import { body, textSans } from '@guardian/src-foundations/typography';
+import { palette, space } from '@guardian/src-foundations';
+import { Button } from '@guardian/src-button';
+import { ButtonLink } from '@guardian/src-link';
+import { css } from '@emotion/core';
+import { OphanComponentEvent } from '../../../types/OphanTypes';
+import {
+    OPHAN_COMPONENT_ARTICLE_COUNT_OPT_OUT_OPEN,
+    OPHAN_COMPONENT_ARTICLE_COUNT_OPT_OUT_CLOSE,
+    OPHAN_COMPONENT_ARTICLE_COUNT_STAY_IN,
+    OPHAN_COMPONENT_ARTICLE_COUNT_OPT_OUT,
+    OPHAN_COMPONENT_ARTICLE_COUNT_STAY_OUT,
+    OPHAN_COMPONENT_ARTICLE_COUNT_OPT_IN,
+} from './utils/ophan';
+
+// --- Styles --- //
+
+const containerStyles = css`
+    ${body.medium({ fontWeight: 'bold' })};
+    font-style: italic;
+`;
+
+const optOutContainer = css`
+    color: ${palette.opinion[400]};
+`;
+
+const articleCountOnHeaderContainerStyles = css`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+`;
+
+const articleCountTextStyles = css`
+    ${textSans.small()};
+    margin-right: ${space[1]}px;
+`;
+
+const articleCountWrapperStyles = css`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-right: ${space[2]}px;
+    justify-content: flex-end;
+`;
+
+const articleCountCtaStyles = css`
+    ${textSans.small({ fontWeight: 'bold' })};
+`;
+
+const articleCountDescriptionTopContainerStyles = css`
+    border-top: 1px solid #000000;
+    border-bottom: 1px solid #000000;
+    margin-top: ${space[4]}px;
+    position: relative;
+`;
+
+const articleCountDescriptionContainer = css`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: ${space[3]}px 0;
+`;
+
+const articleCountBodyTextStyles = css`
+    ${textSans.small()};
+    width: 65%;
+`;
+
+const articleCountCtasContainerStyles = css`
+    display: flex;
+    flex-direction: column;
+    margin-left: auto;
+    justify-content: space-between;
+    height: 60px;
```

Comment:
It might be best here to specify the space between the buttons explicitly. We can use the 'lobotomised owl' selector here: 

```css
> * + * {
    margin-top: ${space[2]}px;
}
```
It looks a little scary but we can break it down! The `>` is just targeting direct children, i.e just the `Button`s, not any of their children. Then the `* + *` is targeting anything that has a 'left sibling', i.e everything that isn't the first child!

Reply:
great! we can probably drop the explicit height now too :)